Marlin/src/module/probe.cpp

```diff
@@ -370,14 +370,22 @@ FORCE_INLINE void probe_specific_action(const bool deploy) {
     BUZZ(100, 698);
 
     PGM_P const ds_str = deploy ? PSTR(MSG_MANUAL_DEPLOY) : PSTR(MSG_MANUAL_STOW);
-    lcd_setstatusPGM(ds_str);
+    // leave the menu if it is displayed to make LCD messsages visible
+    lcd_return_to_status();
+    lcd_setstatusPGM(ds_str, 99);
+    #if HAS_LCD_MENU
+      lcd_quick_feedback(true);
```

Comment:
I don't feel it's appropriate to make a click sound if the user hasn't actually pressed the click-wheel or a keypad button. The concept of adding audio feedback to Marlin exists, but it would have event-appropriate sounds indicating success, failure, completion, etc.

Reply:
oh. ok. I'll remove it - must have been a copy-artefact, sorry

Reply:
sidenote: it's not super-obvious from the method name, what it does :)